File: app/src/main/java/com/hex/evegate/api/StationApi.java
package com.hex.evegate.api;

import com.hex.evegate.api.dto.NowPlayingDto;

import io.reactivex.Observable;
import retrofit2.Response;
import retrofit2.http.GET;

public interface StationApi {

    @GET("nowplaying/1")
    public Observable<Response<NowPlayingDto>> nowPlaying();
}

File: app/src/main/java/com/hex/evegate/api/StationApi.kt
package com.hex.evegate.api

import com.hex.evegate.ui.mvp.model.NowPlayingDto

import kotlinx.coroutines.Deferred
import retrofit2.Response
import retrofit2.http.GET

interface StationApi {

    @GET("nowplaying/1")
    fun nowPlaying(): Deferred<Response<NowPlayingDto>>
}
